catalog/build/intermediate/genomes-from-ncbi.tsv

```diff
@@ -1,773 +1,773 @@
-Pb01	502779	GCF_000150705.2	True	Scaffold		32925248	110.0	1028461.0	12.0	148.0x	43.0	Full annotation	GCA_000150705.2	Paracoccidioides lutzii	1048829	1,131567,2759,33154,4751,451864,4890,716545,147538,716546,147545,451871,33183,299071,38946,1048829,502779	Ascomycota	Eukaryota	Fungi	Ascomycota	Eurotiomycetes	Onygenales	Ajellomycetaceae	Paracoccidioides	Paracoccidioides lutzii	Paracoccidioides lutzii Pb01	https://genome.ucsc.edu/h/GCF_000150705.2	GCA_000150705.2	GCF_000150705.2	https://hgdownload.soe.ucsc.edu/hubs/GCF/000/150/705/GCF_000150705.2/genes/GCF_000150705.2_Paracocci_br_Pb01_V2.ncbiRefSeq.gtf.gz
-NRRL 181	331117	GCF_000149645.3	True	Scaffold		31377515	163.0	2929192.0	4.0		49.5	Full annotation	GCA_000149645.4	Aspergillus fischeri	36630	1,131567,2759,33154,4751,451864,4890,716545,147538,716546,147545,451871,5042,1131492,5052,2720872,36630,331117	Ascomycota	Eukaryota	Fungi	Ascomycota	Eurotiomycetes	Eurotiales	Aspergillaceae	Aspergillus	Aspergillus fischeri	Aspergillus fischeri NRRL 181	https://genome.ucsc.edu/h/GCF_000149645.3	GCA_000149645.4	GCF_000149645.3	https://hgdownload.soe.ucsc.edu/hubs/GCF/000/149/645/GCF_000149645.3/genes/GCF_000149645.3_ASM14964v4.ncbiRefSeq.gtf.gz
-CBS 7966	425265	GCF_000181695.2	True	Contig		8954998					52.0	Full annotation	GCA_000181695.2	Malassezia globosa	76773	1,131567,2759,33154,4751,451864,5204,452284,1538075,162474,742845,55193,76773,425265	Basidiomycota	Eukaryota	Fungi	Basidiomycota	Malasseziomycetes	Malasseziales	Malasseziaceae	Malassezia	Malassezia globosa	Malassezia globosa CBS 7966	https://genome.ucsc.edu/h/GCF_000181695.2	GCA_000181695.2	GCF_000181695.2	https://hgdownload.soe.ucsc.edu/hubs/GCF/000/181/695/GCF_000181695.2/genes/GCF_000181695.2_ASM18169v2.ncbiRefSeq.gtf.gz
-CBS 160.54	1279043	GCF_000365165.1	True	Scaffold		28991111	19.0	3197266.0	4.0	80.0x	54.5	Full annotation	GCA_000365165.2	Cladophialophora carrionii	86049	1,131567,2759,33154,4751,451864,4890,716545,147538,716546,147545,451870,34395,43219,82105,86049,1279043	Ascomycota	Eukaryota	Fungi	Ascomycota	Eurotiomycetes	Chaetothyriales	Herpotrichiellaceae	Cladophialophora	Cladophialophora carrionii	Cladophialophora carrionii CBS 160.54	https://genome.ucsc.edu/h/GCF_000365165.1	GCA_000365165.2	GCF_000365165.1	https://hgdownload.soe.ucsc.edu/hubs/GCF/000/365/165/GCF_000365165.1/genes/GCF_000365165.1_Clad_carr_CBS_160_54_V1.ncbiRefSeq.gtf.gz
-CBS 101466	1220924	GCF_000365145.1	True	Scaffold		28781303	19.0	2379673.0	5.0	132.0x	54.0	Full annotation	GCA_000365145.2	Cyphellophora europaea	293227	1,131567,2759,33154,4751,451864,4890,716545,147538,716546,147545,451870,34395,1341112,226991,293227,122
```

Comment:
The GTF urls didn't change. I don't know what's up with `Eukaryota` missing from the diff.

Reply:
Thanks, will take a look!